pwa/src/templates/templateParts/sourcesForm/SourcesAuthFormTemplate.tsx

```diff
@@ -62,21 +70,53 @@ export const SourcesAuthFormTemplate: React.FC<SourcesAuthFormTemplateProps & Re
                 {...{ register, errors, disabled }}
                 name="password"
                 validation={{ required: true, maxLength: 225 }}
+                disabled={isLoading.sourceForm}
               />
               {errors["password"] && <ErrorMessage message={errors["password"].message} />}
             </FormFieldInput>
           </FormField>
         </>
       );
 
-    case "jwt":
+    case "jwt-HS256":
```

Comment:
I think, instead of writing the JSX _within_ this switch case, we should make separate components (local components that are never exported, at the bottom of this file) and return those when the case is hit, like so:

```
switch (selectedAuth) {
  case "apikey":
    return <AuthApiKey { ...rest } />; // ...rest syntax will be effective here :)
  ...
  default:
    return <></>; // instead of the "none" option, we could have a default (I think, if for any reason, something that is not TSourcesAuthType is passed, this would otherwise break).
}
```